style.css

```diff
@@ -17,28 +94,28 @@ body{
             color:#000000;
             background-color: #b1dfd5;
             width: 100%;
-            height:200px;
+            height:500px;
```

Comment:
I didn't know that "h" was for header. "h1-h5" is the blocks of content on my site. I wanted to group different aspects together. Now I'm thinking that "div" would be better for this.